agenda-webapps/src/main/webapp/vue-app/agenda/components/AgendaEvent.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <v-menu
+      v-model="selectedOpen"
+      :close-on-content-click="false"
+      :activator="selectedElement"
+      offset-x>
+      <v-card
+        color="grey lighten-4"
+        min-width="350px"
```

Comment:
For the sizes, I would suggest to use CSS properties instead

Reply:
done